modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -9048,6 +9048,7 @@ language.it=Italian
 language.iw=Hebrew
 language.ja=Japanese
 language.kk=Kazakh
+language.km_KH=Cambodian
```

Comment:
With the understanding that `km_KH` is the main variant and the only language of Cambodia/Khmer, we will just put this line as 
```
language.km=Cambodian
```

We will only use `km_KH` if there was already one main language `km` and now adding a new variant as `km_KH`.